### PR TITLE
Sound functionality

### PR DESCRIPTION
**Pull request by MonicaStoyanova**

### CODE REVIEW

**Comment by MonicaStoyanova** [comment]
After selection, answer options are blocked
[https://www.awesomescreenshot.com/video/23014046?key=b8db305c002f8c7411527d23570ea64a](url)

**Comment by MonicaStoyanova** [comment]
After selection, answer options are blocked

https://github.com/MonicaStoyanova/who-wants-to-be-a-millionaire/assets/114282401/324e6d10-85b2-47cf-bbe8-20b9f398a07a

File: src/components/Sound/Sound.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      audio.currentTime = 0;
+    };
+  }, [playing, audio]);
+
+  useEffect(() => {
+    audio.addEventListener("ended", () => setPlaying(false));
+    return () => {
+      audio.removeEventListener("ended", () => setPlaying(false));
+    };
+  }, []);
```
Comment (stefanpetkov90) [inline src/components/Sound/Sound.js]: missing dependancy audio

File: src/components/Sound/Sound.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+import styles from "./Sound.module.css";
+import song from "../../assets/sound.mp3";
+
+const useAudio = () => {
+  const [audio] = useState(new Audio(song));
+  const [playing, setPlaying] = useState(true);
+  const toggle = () => setPlaying(!playing);
+
+  useEffect(() => {
+    if (playing) {
```
Comment (stefanpetkov90) [inline src/components/Sound/Sound.js:14]: add the following line above if(playing) to make the audio loop when it ends:

audio.loop = true;

**Pull request merged**